src/html/m_image.cpp

```diff
@@ -757,10 +757,11 @@ TAG_HANDLER_BEGIN(IMG, "IMG,MAP,AREA")
         {
             m_WParser->CloseContainer();
             m_WParser->OpenContainer();
-            wxString tmp;
-            if (tag.GetParamAsString(wxT("NAME"), &tmp))
+            wxString name, id;
+            if (tag.GetParamAsString(wxT("NAME"), &name) || tag.GetParamAsString(wxT("ID"), &id))
             {
-                wxHtmlImageMapCell *cel = new wxHtmlImageMapCell( tmp );
+                wxHtmlImageMapCell *cel = new wxHtmlImageMapCell( name );
+                cel->SetId(tag.GetParam(wxT("id")));
```

Comment:
Isn't this the same thing as
```suggestion
                cel->SetId(id));
```
? If not, it would merit a comment explaining why not because this doesn't seem obvious to me...

Reply:
added a comment